corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -701,20 +708,47 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
      * @param {Object} json - The JSON returned from touchforms to represent a Form
      * @param {Object} parent - The object's parent. Either a Form, Group, or Repeat.
      */
-    function GroupedQuestionTileRow(json, parent) {
+    function GroupedElementTileRow(json, parent) {
         var self = this;
         self.parent = parent;
         Container.call(self, json);
 
+        self.isRepetition = parent instanceof Repeat;
```

Comment:
i don't think you need this. But instead, [this](https://github.com/dimagi/commcare-hq/blob/96d92204f04a265cecabbc34880c33e70ac79a90/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L581C17-L581C17) should be changed to `parent.parent instanceof Repeat;`

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/dbab40c2c0c15313bc355da9910ac8ee96dcb24d